torch/_inductor/graph.py

```diff
@@ -211,11 +218,25 @@ def __init__(
         self.sizevars = SizeVarAllocator(shape_env)
         self.graph_inputs: Dict[str, TensorBox] = {}
         self.graph_inputs_original: Dict[str, InputBuffer] = {}
-        self.device_types: Set[str] = set()
-        self.device_idxs: Set[int] = set()
+        self.device_types: Set[str] = (
+            const_module.device_types if const_module else set()
+        )
+        self.device_idxs: Set[int] = const_module.device_idxs if const_module else set()
         self.cuda = False
         self.buffers: List[ir.Buffer] = []
-        self.constants: Dict[str, torch.Tensor] = {}
+        self.const_output_index: Dict[str, int] = (
+            const_output_index if const_output_index else {}
+        )
+        self.const_kernels: Set[str] = (
+            const_module.wrapper_code.src_to_kernel.values() if const_module else set()
```

Comment:
This is never used.

Reply:
This is used in `wrapper.py`. This is used in the case that we might have duplicated kernels between constant module and main module.